merged_fs.go

```diff
@@ -509,6 +509,59 @@ func (m *MergedFS) Open(path string) (fs.File, error) {
 	return fB, nil
 }
 
+// ReadFile reads the named file and returns the contents.
+// A successful call returns err == nil, not err == EOF.
+// Because ReadFile reads the whole file, it does not treat an EOF from Read
+// as an error to be reported.
+// This implementation was mostly copied from the golang stdlib os.ReadFile.
+// This is being added so that we fulfill the io/fs.ReadFileFS interface.
+// https://pkg.go.dev/io/fs#ReadFileFS
+func (m *MergedFS) ReadFile(name string) ([]byte, error) {
+	f, err := m.Open(name)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close() // ignore error
+
+	if true { // safer method in case the stdlib changes
```

Comment:
Can we just keep the call to `io.ReadAll(f)` and remove the remaining dead code?